_posts/2016/2016-11-28-xp-days-germany-2016.md

```diff
@@ -0,0 +1,120 @@
+In this context also several product owners, scrum masters and innovation evangelists had the courage to participate and contribute.
+
+# The schedule
+
+This time the three-day-conference with around 170 participants was even accompanied by a [Devoxx4Kids](http://www.devoxx4kids-hamburg.de) event, so that theoretically the whole geek family could join.
+[Sunday](http://www.xpdays.de/2016/downloads/community-day/OpenSpaceSonntag.pdf) was assembled in an open space format with a session marketplace in the morning and after the lunch break.
+The sessions varied from solving coding katas to sharing knowledge and exchanging experiences.
+Monday and Tuesday in contrast provided a regular [conference schedule](http://www.xpdays.de/2016/programm/) with over 32 talks, 8 workshops, 11 lightning talks, 2 keynotes and some flexible slots – [Lean Coffee](http://german.leancoffee.org/) and an additional smaller open space.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Keynote-Feeling bei den XP Days <a href="https://twitter.com/hashtag/developer?src=hash">#developer</a> <a href="https://twitter.com/hashtag/agile?src=hash">#agile</a> <a href="https://twitter.com/hashtag/programming?src=hash">#programming</a> <a href="https://twitter.com/hashtag/conference?src=hash">#conference</a> <a href="https://twitter.com/hashtag/xdde16?src=hash">#xdde16</a> <a href="https://twitter.com/xd_de">@xd_de</a> <a href="https://t.co/wt0L6pnJyC">pic.twitter.com/wt0L6pnJyC</a></p>&mdash; Benjamin Nothdurft (@dataduke) <a href="https://twitter.com/dataduke/status/800710674884141056">November 21, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# Our Docker talk and workshop
+
+After a short pitch in the opening talk we could gather more than 30 interested conference visitors to join our session: "[Welcome to Dockerland: Best Practises for Kickstarters and Tools by Example](http://www.xpdays.de/2016/sessions/116-welcome-to-dockerland-best-practices-for-kickstarters-tools-by-example)".
+Here we explained the various parts of the Docker platform and showed best practices for writing Dockerfiles.
+Briefly we could also introduce some examples with [Spring Boot](https://spring.io/guides/gs/spring-boot-docker/) and Gradle, Node.js and MongoDB, Logstash and Elasticsearch, Docker image tag strategies with build servers as well as multi-container apps with Docker Compose.
+At the end we got some very positive feedback for our [pictographic slides](https://speakerdeck.com/dataduke/welcome-to-dockerland-xdde16).
+
+<script async class="speakerdeck-embed" data-id="7aacbd41660c4f86b2e1b5fff3b5afac" data-ratio="1.37081659973226" src="//speakerdeck.com/assets/embed.js"></script>
+
+For a more practical approach we invited all participants to join our hands-on workshop in the afternoon which offered [5 different tutorials](http://training.docker.com/category/docker-mentor-week) for all skill levels.
```

Comment:
Two commas missing:
For a more practical approach, we invited... 
..in the afternoon, which offered...